test/column/formatter.test.tsx

```diff
@@ -110,3 +110,41 @@ describe('Custom formatter component', () => {
     });
   });
 });
+
+describe('formatter focus', () => {
+  const columns: readonly Column<Row>[] = [{ key: 'id', name: 'ID' }];
+  function FormatterTest() {
+    const [inputValue, setInputValue] = useState('');
+    const [rows, setRows] = useState((): readonly Row[] => [{ id: 1 }]);
+
+    function onChange(event: React.ChangeEvent<HTMLInputElement>) {
+      setInputValue(event.target.value);
+      setRows([{ id: 2 }]);
+    }
+
+    return (
+      <StrictMode>
+        <input aria-label="outer-input" value={inputValue} onChange={onChange} />
+        <DataGrid
+          columns={columns}
+          rows={rows}
+          onRowsChange={setRows}
+          rowKeyGetter={(row) => row.id}
+        />
+      </StrictMode>
+    );
+  }
+
+  it.skip('should not steal focus back to the cell if the focus is outside the grid and formatter is recreated', async () => {
```

Comment:
With the describe name it becomes

>formatter focus should not steal focus back to the cell if the focus is outside the grid and formatter is recreated

the describe name should be changed, or removed, wdyt?